src/__tests__/PostProtectedIndex.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import PostProtectedIndex from '../pages/PostProtectedIndex';
+import mockPosts from '../mockPosts'
+import { BrowserRouter } from 'react-router-dom';
+
+const currentUser = { id: 1};
```

Comment:
recommend making sure that your mock data is a reflecting of your data that you receive from the backend
Consider importing your mockUser file